src/main/asciidoc/batch/file-ingest-sftp/main.adoc

```diff
@@ -1,513 +1,246 @@
-services:
-  - mysql
-  - redis
-----
+Build the app:
 +
-If your Cloud Foundry installation is behind a firewall, you may need to install the stream apps used in this sample in your internal Maven repository and https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#configuration-maven[configure] the server to access that repository.
-. Once you have successfully executed `cf push`, verify the dataflow server is running
-+
-
 ```
-$ cf apps
-Getting apps in org [your-org] / space [your-space] as user...
-OK
-
-name                 requested state   instances   memory   disk   urls
-dataflow-server      started           1/1         1G       1G     dataflow-server.app.io
+$./mvnw clean package
 ```
-
-. Notice that the `dataflow-server` application is started and ready for interaction via the url endpoint
-
-. Connect the `shell` with `server` running on Cloud Foundry, e.g., `http://dataflow-server.app.io`
+. Register the jar
 +
-```
-$ cd <PATH/TO/SPRING-CLOUD-DATAFLOW-SHELL-JAR>
-$ java -jar spring-cloud-dataflow-shell-<VERSION>.jar
-
-  ____                              ____ _                __
- / ___| _ __  _ __(_)_ __   __ _   / ___| | ___  _   _  __| |
- \___ \| '_ \| '__| | '_ \ / _` | | |   | |/ _ \| | | |/ _` |
-  ___) | |_) | |  | | | | | (_| | | |___| | (_) | |_| | (_| |
- |____/| .__/|_|  |_|_| |_|\__, |  \____|_|\___/ \__,_|\__,_|
-  ____ |_|    _          __|___/                 __________
- |  _ \  __ _| |_ __ _  |  ___| | _____      __  \ \ \ \ \ \
- | | | |/ _` | __/ _` | | |_  | |/ _ \ \ /\ / /   \ \ \ \ \ \
- | |_| | (_| | || (_| | |  _| | | (_) \ V  V /    / / / / / /
- |____/ \__,_|\__\__,_| |_|   |_|\___/ \_/\_/    /_/_/_/_/_/
-
-
-Welcome to the Spring Cloud Data Flow shell. For assistance hit TAB or type "help".
-server-unknown:>
-```
+If running in Cloud Foundry, the resulting executable jar file must be available in a location that is accessible to your Cloud Foundry instance, such as an HTTP server or Maven repository.
+If running on a local server:
 +
 ```
-server-unknown:>dataflow config server http://dataflow-server.app.io
-Successfully targeted http://dataflow-server.app.io
-dataflow:>
+dataflow>app register --name sftp --type source --uri file:<project-directory>/sftp/apps/sftp-dataflow-source-kafka/target/sftp-dataflow-source-kafka-X.X.X.jar --force
 ```
+. Run the Demo
 +
-
-
-===== Building and Running the Demo
-
-. Build the demo JAR
+Follow the instructions for building and running the main SFTP File Ingest demo, for your preferred platform, up to the `Add Data Step`.
+If you have already completed the main exercise, restore the data to its initial state, and redeploy the stream:
 +
-Building upon the code in `batch/file-ingest-sftp`, in this demo we utilize https://cloud.spring.io/spring-cloud-connectors/[Spring Cloud Connectors] to automatically bind Cloud Foundry services such as MySQL and Redis.
+* Clean the data directories
```

Comment:
Maybe put the directories in the paranthesis (/eg: `tmp/remotre-files`; `/tmp/local-files`)